Update license statement in README

Kasijjuf opened the pull request.

Kasijjuf committed:
Correct typo in README

while maintaining the apparent 70 column text justification

parazyd commented:
For what it's worth, the whole thing could be changed because that is not the address of the FSF anymore. We should rather use the new header which includes the URL: http://www.gnu.org/licenses/

jaromil commented:
Oh good that it was updated without address indeed. We will use that rather than merge this
```
This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
...it will need to be updated in a lot of places in our repositories...

Kasijjuf committed:
Update license statement in README

to the latest version published by the FSF at
<https://www.gnu.org/licenses/gpl-howto.html>.
Also formatted to maintain the apparent
70 column text justification.
